### ISSUE TITLE
AW Admin garbles non-english characters

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/HPI#2243]
See the screenshots below.  AW admin worked with non-english characters at some point - I was able to create the `TS Non-English` page set on edge1/2.  However, it seems like something has changed and the admin is now garbling characters.  See below:

Spanish (before saving):
![image](https://user-images.githubusercontent.com/1377228/91062899-e11f6980-e5f2-11ea-85e4-4375fab91911.png)

Spanish (after saving):
![image](https://user-images.githubusercontent.com/1377228/91062925-ec729500-e5f2-11ea-8b2e-f9b439398330.png)

Chinese (before saving):
![image](https://user-images.githubusercontent.com/1377228/91063024-0f04ae00-e5f3-11ea-8ee7-6d1f94250991.png)

Chinese (after saving):
![image](https://user-images.githubusercontent.com/1377228/91063052-17f57f80-e5f3-11ea-82a5-7d2d6b263508.png)

Related commits: 
- https://github.com/tsgrp/ActiveWizard/issues/256 - this fixed non-english, latin-based languages (Spanish, French, etc) for AW forms
- #2112 - this fixed double byte languages (Chinese, Japanese, etc) for AW forms

**Comment by gsteimer** [comment]
Not sure if it would be causing this, but I'm pretty sure that both edge1 and edge2 Tomcats are set up with this as well:

https://github.com/tsgrp/HPI/wiki/Installation-Requirements#utf-8-encoding

**Comment by gsteimer** [comment]
Turns out I was wrong in the last comment - none of our environments had the proper `server.xml` and `web.xml` settings in the `${tomcat.home}/conf` folder.  Adding these in solved the issue. 

One note - in some cases (but not all) I had to edit the form questions/answers and re-save with the correct values.  For example:

- edge1 always was garbled.  I had to resave both Spanish and Chinese
- edge2 was fine with Spanish after restarting.  However, I had to re-save the Chinese page with the correct values.
